cmd/config.go

```diff
@@ -45,7 +46,8 @@ func createConfig(home string, debug bool) error {
 	return nil
 }
 
-func overwriteConfig(home string, cfg *Config) error {
+func overwriteConfig(cfg *Config) error {
+	home := viper.GetString("home")
```

Comment:
Is relying on viper OK here?

Reply:
It works - if that's what you're asking 😃 . I suppose it's not the cleanest. We could put `home` onto the Config object. 

Seemed like a lot of work to pass in every time. Can revert back to that though if you think it's better.